sympy/logic/boolalg.py

```diff
@@ -1304,6 +1304,87 @@ def append_symbol(arg, symbols):
             for c in clauses]
 
 
+def term_to_integer(term):
+	"""
+	Constructs an integer from its base 2 digits.
+
+	Examples
+	========
+
+	>>> from sympy.logic.boolalg import term_to_integer
+	>>> term_to_integer([1, 0, 1, 1])
+	11
+	>>> terms = [[0, 0, 0, 1], [0, 0, 1, 1],
+	...			[0, 1, 1, 1], [1, 0, 1, 1], [1, 1, 1, 1]]
+	>>> map(term_to_integer, terms)
+	[1, 3, 7, 11, 15]
+
+	"""
+
+	term = list(term)
+	return int(''.join(map(str, term)), 2)
+
+
+def integer_to_term(i, n_bits=0):
+	"""
+	Gives a list of the base 2 digits in the integer i.
+	If additional n_bits argument is passed,
+	integer_to_term() extends list of digits with zeros.
+
+	Examples
+	========
+
+	>>> from sympy.logic.boolalg import integer_to_term
+	>>> integer_to_term(14)
+	[1, 1, 1, 0]
+	>>> integer_to_term(7, 5)
+	[0, 0, 1, 1, 1]
+	>>> truthtable = [1, 0, 0, 1, 1, 1, 0, 1]
+	>>> minterms = []
+	>>> n_bits = len(truthtable).bit_length() - 1
+	>>> for i, value in enumerate(truthtable):
+	...			if value:
+	...				minterms.append(integer_to_term(i, n_bits))
+	>>> minterms
+	[[0, 0, 0], [0, 1, 1], [1, 0, 0], [1, 0, 1], [1, 1, 1]]
+
+	"""
+	n = i.bit_length() if i else 1
+	prefix = [0] * (n_bits - n if n_bits else 0)
+	return prefix + map(int, list('{0:b}'.format(i)))
+
+
+def truth_table(expr, variables):
+	"""
+	Gives a list of truth values of the boolean expression.
+
+	Examples
+	========
+	>>> from sympy.logic.boolalg import truth_table
+	>>> truth_table('a&b | b&c | c&a', ['a', 'b', 'c'])
+	[0, 0, 0, 1, 0, 1, 1, 1]
```

Comment:
> Also, integer_to_term can be used to obtain truth values

I followed you to here, but this example lost me. What is n used for? Is there a standard order for the 16 truth functions? Anyway, this isn't necessary (to me) since what preceded was clear.